### ISSUE TITLE
Crash when exporting a "local copy" image.

### ISSUE THREAD

**Issue by cldershem** [darktable-org/darktable#6120]
**Describe the bug**
Attempting to export an "local copy" image causes darktable to hang and eventually crash.

**To Reproduce**
1. Open a film roll in lighttable
2. Select a few images on an external drive
3. Make a local copy
4. Disconnect external drive
5. Open a local copy in darkroom
6. Attempt to export the image to disk (jpg 95
7. Darktable will freeze

**Expected behavior**
I expect darktable to export and write a file to disk.

**Platform (please complete the following information):**
- Ubuntu 20.04
- Dell XPS 13 7390
- Intel(R) UHD Graphics
- this is darktable 3.2.1
    compile options:
      bit depth is 64 bit
      normal build
      SSE2 optimized codepath enabled
      OpenMP support enabled
      OpenCL support enabled
      Lua support enabled, API version 6.0.0
      Colord support enabled
      gPhoto2 support enabled
      GraphicsMagick support enabled
      ImageMagick support disabled
      OpenEXR support enabled

**Additional context**
- The image is RAF file from a Fuji X-T2.
- No other darktable versions available.

**Comment by TurboGit** [comment]
I cannot reproduce. This is maybe because dt has not been restarted and a file was still opened for a picture now off-line. So can you test this : 
- after step 3 quit dt
- restart it after step 4.

**Comment by maboleth** [comment]
Cannot reproduce also. Local copy does well, without any issues, both on regular 3.2 and master git.

**Comment by cldershem** [comment]
Ah. You're right. That works. Maybe this should be stated explicitly in the docs. I tired this several different times, but it never occured to me that I'd need to restart after disconnecting my drive. It makes total sense, but I didn't think of it.

**Comment by matt-maguire** [comment]
I couldn't get 3.2.1 to crash at all on Manjaro Linux, regardless whether I cleanly or uncleanly removed the drive while darktable was running.

**Comment by cldershem** [comment]
I really did reproduce this several times before opening an issue, I swear! ha.

Feel free to close for now. I have another issue that may be related, but haven't had time to test that theory yet. If it happens again I can reopen it with more details.

**Comment by johnny-bit** [comment]
> Feel free to close for now.

OK then :)
